src/components/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <nav>
+      <h2 className="Logo">Math Magician</h2>
+      <ul>
+        <li>
```

Comment:
- [optional] you can add more style to the nav bar to the extent of it indicating the current page on the page link(for instance underlining the current page link). Great job